pkg/kv/kvserver/replica_proposal_buf.go

```diff
@@ -588,7 +588,7 @@ func (b *propBuf) FlushLockedWithRaftGroup(
 		return 0, firstErr
 	}
 
-	propErr, _ := proposeBatch(ctx, b.p, raftGroup, b.p.getReplicaID(), ents, admitHandles, buf[firstProp:nextProp])
+	propErr, _ := proposeBatch(ctx, b.p, raftGroup, ents, admitHandles, buf[firstProp:nextProp])
```

Comment:
Do any of the callers still need the second returned parameter?
Tests, potentially? But I don't see any. Consider removing it then.

Reply:
I don't know how far up the stack this error can make, whether it can make into text logs or get outside the process. It can contain user data, right? So that's a bit sensitive. Maybe err on the safe side, and include only necessary info in the error message. See a similar `panic` in `maybeDeductFlowTokens`.